cmd/nvidia-dra-plugin/main.go

```diff
@@ -183,19 +183,24 @@ func StartPlugin(ctx context.Context, config *Config) error {
 		return fmt.Errorf("path for cdi file generation is not a directory: '%v'", config.flags.cdiRoot)
 	}
 
-	driver, err := NewDriver(ctx, config)
-	if err != nil {
-		return err
-	}
-
 	sigc := make(chan os.Signal, 1)
```

Comment:
Not specific to this PR, but I think in the controller component we use `sigs` here? I suppose `sigc` is Sig(nal) C(hannel)? Do we want to update for consistency?

Reply:
I went back and looked and it seems we actually do use `sigs` everywhere -- not sure why I thought it was `sigc`. Updated.